What does the config file for Burrow 1.0RC look like? panic No Zookeeper servers specified

Without knowing Viper I didn't realize that .cfg or no extension will not work.  I strace'd and it's looking for one of a few extensions, toml seemed like the best candidate since it looks like the burrow.cfg form the pre-1.0 days, but it still doesn't seem to work. 

```
[pid 11975] stat("/home/dennis/burrow.json", 0xc420081148) = -1 ENOENT (No such file or directory) <0.000026>
[pid 11975] stat("/home/dennis/burrow.toml", 0xc420081218) = -1 ENOENT (No such file or directory) <0.000027>
[pid 11975] stat("/home/dennis/burrow.yaml", 0xc4200812e8) = -1 ENOENT (No such file or directory) <0.000024>
[pid 11975] stat("/home/dennis/burrow.yml", 0xc4200813b8) = -1 ENOENT (No such file or directory) <0.000025>
[pid 11975] stat("/home/dennis/burrow.properties", 0xc420081488) = -1 ENOENT (No such file or directory) <0.000025>
[pid 11975] stat("/home/dennis/burrow.props", 0xc420081558) = -1 ENOENT (No such file or directory) <0.000023>
[pid 11975] stat("/home/dennis/burrow.prop", 0xc420081628) = -1 ENOENT (No such file or directory) <0.000025>
[pid 11975] stat("/home/dennis/burrow.hcl", 0xc4200816f8) = -1 ENOENT (No such file or directory) <0.000025>
```

```
$ ./burrow_beta
Reading configuration from .
{"level":"info","ts":1510396387.319005,"msg":"Started Burrow"}
{"level":"info","ts":1510396387.3191428,"msg":"configuring","type":"coordinator","name":"zookeeper"}
{"level":"panic","ts":1510396387.3191752,"msg":"No Zookeeper servers specified"}
panic: No Zookeeper servers specified [recovered]
	panic: No Zookeeper servers specified [recovered]
	panic: No Zookeeper servers specified

goroutine 1 [running]:
main.handleExit()
	/Users/dennis/go/src/github.com/linkedin/Burrow/main.go:42 +0x293
panic(0x8f68e0, 0xc420254890)
	/usr/local/go/src/runtime/panic.go:491 +0x283
go.uber.org/zap/zapcore.(*CheckedEntry).Write(0xc4200be420, 0x0, 0x0, 0x0)
	/Users/dennis/go/src/go.uber.org/zap/zapcore/entry.go:229 +0x52e
go.uber.org/zap.(*Logger).Panic(0xc42008cc60, 0x9e04f3, 0x1e, 0x0, 0x0, 0x0)
	/Users/dennis/go/src/go.uber.org/zap/logger.go:225 +0x7f
github.com/linkedin/Burrow/core.configureCoordinators.func1(0xc42008c7e0)
	/Users/dennis/go/src/github.com/linkedin/Burrow/core/burrow.go:89 +0x88
panic(0x8f68e0, 0xa38a20)
	/usr/local/go/src/runtime/panic.go:491 +0x283
github.com/linkedin/Burrow/core/internal/zookeeper.(*Coordinator).Configure(0xc42024edc0)
	/Users/dennis/go/src/github.com/linkedin/Burrow/core/internal/zookeeper/coordinator.go:48 +0x2a5
github.com/linkedin/Burrow/core.configureCoordinators(0xc42008c7e0, 0xc82860, 0xc42024edc0, 0xc82820, 0xc42024dd40, 0xc82460, 0xc4202602c0, 0xc824e0, 0xc420260320, 0xc825e0, ...)
	/Users/dennis/go/src/github.com/linkedin/Burrow/core/burrow.go:96 +0x9c
github.com/linkedin/Burrow/core.Start(0xc42008c7e0, 0xc42008cd20, 0x3)
	/Users/dennis/go/src/github.com/linkedin/Burrow/core/burrow.go:124 +0x324
main.main()
	/Users/dennis/go/src/github.com/linkedin/Burrow/main.go:93 +0x457
```

This is the zookeeper section:
```
[zookeeper]
hostname="zookeeper.dns"
port=2181
timeout=6
lock-path="/burrow/notifier"
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,10 +1,13 @@
-## X.Y.Z (TBD)
+## 1.0.0 (TBD)
 
 Features:
-  - Added request info to HTTP responses (#64 and #45)
+  - Code overhaul - more modular and now with tests
+  - Actual documentation (godoc)
+  - Support for topic deletion in Kafka clusters
+  - Removed Slack notifier in favor of just using the HTTP notifier
 
 Bugfixes:
-  - Fix an issue where maxlag partition is selected badly
+  - Too many to count
 
 ## 0.1.1 (2016-05-01)
 

--- a/README.md
+++ b/README.md
@@ -16,19 +16,19 @@ Burrow is a monitoring companion for [Apache Kafka](http://kafka.apache.org) tha
 ### Prerequisites
 Burrow is written in Go, so before you get started, you should [install and set up Go](https://golang.org/doc/install).
 
-If you have not yet installed the [Go Package Manager](https://github.com/pote/gpm), please go over there and follow their short installation instructions. GPM is used to automatically pull in the dependencies for Burrow so you don't have to chase them all down.
+If you have not yet installed the [Go Dependency Management Tool](https://github.com/golang/dep), please go over there and follow their short installation instructions. dep is used to automatically pull in the dependencies for Burrow so you don't have to chase them all down.
 
 ### Build and Install
 ```
 $ go get github.com/linkedin/Burrow
 $ cd $GOPATH/src/github.com/linkedin/Burrow
-$ gpm install
+$ dep ensure
 $ go install
 ```
 
 ### Running Burrow
 ```
-$ $GOPATH/bin/Burrow --config path/to/burrow.cfg
+$ $GOPATH/bin/Burrow --config-dir /path/containing/config
 ```
 
 ### Using Docker
@@ -55,7 +55,7 @@ Install Docker Compose and then:
 For information on how to write your configuration file, check out the [detailed wiki](https://github.com/linkedin/Burrow/wiki)
 
 ## License
-Copyright 2015 LinkedIn Corp. Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+Copyright 2017 LinkedIn Corp. Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
 You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
 
 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR

--- a/config/burrow.cfg
+++ b/config/burrow.cfg
@@ -1,88 +1,61 @@
 [general]
-logdir=log
-logconfig=config/logging.cfg
-pidfile=burrow.pid
-client-id=burrow-lagchecker
-group-blacklist=^(console-consumer-|python-kafka-consumer-).*$
-#group-whitelist=^(my-important-consumer).*$
+pidfile=log/burrow.pid
+
+[logging]
+filename=log/burrow.log
 
 [zookeeper]
-hostname=zkhost01.example.com
-hostname=zkhost02.example.com
-hostname=zkhost03.example.com
-port=2181
+servers=[ "zkhost01.example.com:2181", "zkhost02.example.com:2181", "zkhost03.example.com:2181" ]
 timeout=6
-lock-path=/burrow/notifier
+root-path=/burrow
 
-[kafka "local"]
-broker=kafka01.example.com
-broker=kafka02.example.com
-broker=kafka03.example.com
-broker=kafka04.example.com
-broker=kafka05.example.com
-broker-port=10251
-zookeeper=zkhost01.example.com
-zookeeper=zkhost02.example.com
-zookeeper=zkhost03.example.com
-zookeeper-port=2181
-zookeeper-path=/kafka-cluster
-offsets-topic=__consumer_offsets
+[client-profile.test]
+client-id="burrow-test"
+kafka-version="0.10.0"
 
-[storm "local"]
-zookeeper=zkhost01.example.com
-zookeeper=zkhost02.example.com
-zookeeper=zkhost03.example.com
-zookeeper-port=2181
-zookeeper-path=/kafka-cluster/stormconsumers
+[cluster.local]
+class-name="kafka"
+servers=[ "kafka01.example.com:10251", "kafka02.example.com:10251", "kafka03.example.com:10251" ]
+client-profile="test"
+topic-refresh=120
+offset-refresh=30
 
-[tickers]
-broker-offsets=60
+[consumer.local]
+class-name="kafka"
+cluster="local"
+servers=[ "kafka01.example.com:10251", "kafka02.example.com:10251", "kafka03.example.com:10251" ]
+client-profile="test"
+group-blacklist="^((console-consumer-|python-kafka-consumer-|quick-).*$"
+group-whitelist=""
 
-[lagcheck]
-intervals=10
+[consumer.local_zk]
+class-name="kafka_zk"
+cluster="local"
+servers=[ "zk01.example.com:2181", "zk02.example.com:2181", "zk03.example.com:2181" ]
+zookeeper-path="/kafka-cluster"
+zookeeper-timeout=30
+group-blacklist="^((console-consumer-|python-kafka-consumer-|quick-).*$"
+group-whitelist=""
+
+[httpserver.default]
+address=":8000"
+
+[storage.default]
+class-name="inmemory"
+workers=20
+intervals=15
 expire-group=604800
+min-distance=1
 
-[httpserver]
-server=on
-port=8000
-; Alternatively, use listen (cannot be specified when port is)
-; listen=host:port
-; listen=host2:port2
-; For alternative Access Control, set this to the appropriate header. For example, a possibly unsecure option may be
-; access-control-allow-origin=*
-
-[smtp]
-server=mailserver.example.com
-port=25
-from=burrow-noreply@example.com
-template=config/default-email.tmpl
-
-[emailnotifier "bofh@example.com"]
-group=local,critical-consumer-group
-group=local,other-consumer-group
+[notifier.default]
+class-name="http"
+url-open="http://someservice.example.com:1467/v1/event"
 interval=60
-
-[notify]
-interval=10
-
-[httpnotifier]
-url=http://notification.server.example.com:9000/v1/alert
-interval=60
-extra=app=burrow
-extra=tier=STG
-template-post=config/default-http-post.tmpl
-template-delete=config/default-http-delete.tmpl
 timeout=5
 keepalive=30
-
-[slacknotifier]
-enable=true
-url=https://hooks.slack.com/services/xxxx/xxxxxxxxxx
-group=local,critical-consumer-group
-group=local,other-consumer-group
-threshold=0
-channel="#general"
-username=burrower
-interval=5
-timeout=5
-keepalive=30
+extras={ api_key="REDACTED", app="burrow", tier="STG", fabric="mydc" }
+template-open="conf/default-http-post.tmpl"
+template-close="conf/default-http-delete.tmpl"
+method-close="DELETE"
+send-close=true
+threshold=1

--- a/config/default-email.tmpl
+++ b/config/default-email.tmpl
@@ -1,17 +1,11 @@
-From: {{.From}}
-To: {{.To}}
 Subject: [Burrow] Kafka Consumer Lag Alert
 
 The Kafka consumer groups you are monitoring are currently showing problems. The following groups are in a problem state (groups not listed are OK):
 
-{{range .Results}}{{if or (ne 0 .Status) (ne 5 .Status)}}Cluster:  {{.Cluster}}
-Group:    {{.Group}}
-Status:   {{if eq 1 .Status}}WARNING{{else if eq 2 .Status}}ERROR{{end}}
-Complete: {{.Complete}}
-Errors:   {{len .Partitions}} partitions have problems
-{{range .Partitions}}          {{if eq 1 .Status}} WARN{{else if eq 2 .Status}}  ERR{{else if eq 3 .Status}} STOP{{else if eq 4 .Status}}STALL{{end}} {{.Topic}}:{{.Partition}} ({{.Start.Timestamp}}, {{.Start.Offset}}, {{.Start.Lag}}) -> ({{.End.Timestamp}}, {{.End.Offset}}, {{.End.Lag}})
+Cluster:  {{Result..Cluster}}
+Group:    {{.Result.Group}}
+Status:   {{if eq 1 .Result.Status}}OK{{else if eq 1 .Result.Status}}WARNING{{else if eq 2 .Result.Status}}ERROR{{end}}
+Complete: {{.Result.Complete}}
+Errors:   {{len .Result.Partitions}} partitions have problems
+{{range .Result.Partitions}}          {{if eq 1 .Status}}   OK{{if eq 2 .Status}} WARN{{else if eq 2 .Status}}  ERR{{else if eq 3 .Status}} STOP{{else if eq 4 .Status}}STALL{{end}} {{.Topic}}:{{.Partition}} ({{.Start.Timestamp}}, {{.Start.Offset}}, {{.Start.Lag}}) -> ({{.End.Timestamp}}, {{.End.Offset}}, {{.End.Lag}})
 {{end}}{{end}}
-
-----------------------------------------------------------------------
-{{end}}
-

--- a/config/logging.cfg
+++ b/config/logging.cfg
@@ -1,8 +1,0 @@
-<seelog minlevel="info">
-  <outputs formatid="main">
-    <rollingfile type="date" filename="log/burrow.log" datepattern="2006-02-01-15" maxrolls="168" />
-  </outputs>
-  <formats>
-    <format id="main" format="%Date(2006-01-02 15:04:05) [%LEVEL] %Msg%n"/>
-  </formats>
-</seelog>

